R/reportEmi.R

```diff
@@ -1769,6 +1810,24 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL,
                # total DACCS
                setNames(-out[, , "Carbon Management|Storage|+|DAC (Mt CO2/yr)"],
                         "Emi|CO2|CDR|DACCS (Mt CO2/yr)"),
+
+               # gross OAE
+               setNames(vm_emiCdrTeDetail[, , "oae_ng"] * GtC_2_MtCO2,
+                        "Emi|CO2|CDR|OAE|+|Ocean Uptake, traditional calciner (Mt CO2/yr)"),
+               setNames(vm_emiCdrTeDetail[, , "oae_el"] * GtC_2_MtCO2,
```

Comment:
these two should also get a "+" instead of the "," so they add up to "Ocean Uptake"